all_seaing_driver/all_seaing_driver/thrust_commander.py

```diff
@@ -47,12 +61,74 @@ def back_right_cb(self, msg: Int64):
 
     def back_left_cb(self, msg: Int64):
         self.send_pwm(self.back_left_port, msg.data)
+    
+    def get_time(self):
+        """
+        Get the current time. Written into a function because it's annoying to write every time.
+        """
+        return self.get_clock().now()
+    
+    def turn_off_thrusters(self):
+        """
+        Turn off all thrusters.
+        """
+        self.send_pwm(self.front_right_port, 1500)
+        self.send_pwm(self.front_left_port, 1500)
+        self.send_pwm(self.back_right_port, 1500)
+        self.send_pwm(self.back_left_port, 1500)
 
     def send_pwm(self, channel, value):
         self.get_logger().info(f"Sending PWM value {value} to channel {channel}")
         return self.proxy.call_async(
             CommandLong.Request(command=183, param1=float(channel), param2=float(value))
         )
+    
+    def get_thrust_values(self, tx, ty, tn):
```

Comment:
This is from xdrive controller but isn't used in thrust_commander. Can u delete this?